app/src/main/java/se/hellsoft/android/instantsearchdemo/SearchViewModel.kt

```diff
@@ -31,13 +30,13 @@ class SearchViewModel(
 
     @ExperimentalCoroutinesApi
     @VisibleForTesting
-    internal val queryChannel = BroadcastChannel<String>(Channel.CONFLATED)
+    internal val queryChannel = Channel<String>(Channel.CONFLATED)
 
     @FlowPreview
     @ExperimentalCoroutinesApi
     @VisibleForTesting
     internal val internalSearchResult = queryChannel
-        .asFlow()
+        .consumeAsFlow()
```

Comment:
💭 I'm really hoping this will be consistent once the APIs are final. For now we need this change here as well.